05_stringer/runnerdave/main_test.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
+	actual := strconv.Quote(buf.String())
+	t.Logf("expected:%s", expected)
```

Comment:
More of a question/comment than a change, but in this test you log the expected and actual always, and maybe an error, on your other test you only log when there's an error. Expending this out if you have a lot of tests, would it get a bit noisy to log during success criteria?

Reply:
hi @brnbke this reflects my TDD approach, note that the logging is only in the single test not on the table one, because I write a failing test and then write the function to pass it, if I added logging in the other test I agree the noise would not be useful at all.